frontend/packages/ceph-storage-plugin/integration-tests/views/multiple-pool.view.ts

```diff
@@ -0,0 +1,90 @@
+import * as sideNavView from '@console/internal-integration-tests/views/sidenav.view';
+import * as crudView from '@console/internal-integration-tests/views/crud.view';
+import { click } from '@console/shared/src/test-utils/utils';
+import { $, ExpectedConditions as until, browser, by, element } from 'protractor';
+import { SECOND, NS } from '../utils/consts';
+import { StoragePoolKind } from 'packages/ceph-storage-plugin/src/types';
+import { sendKeys } from '../utils/helpers';
+
+export const poolMessage = {
+  PROGRESS:
+    'The creation of an OCS storage cluster is still in progress or have failed, please try again after the storage cluster is ready to use.',
+  POOL_START: 'Pool sc-pool5 creation in progress',
+  POOL_TIMEOUT:
+    'Pool sc-pool creation timed out. Please check if ocs-operator and rook operator are running',
+  POOL_DUPLICATED: 'cephblockpools.ceph.rook.io "sc-pool" already exists',
+};
+
+export const poolData: StoragePoolKind = {
+  apiVersion: 'ceph.rook.io/v1',
+  kind: 'CephBlockPool',
+  metadata: {
+    name: 'foo',
+    namespace: NS,
+  },
+  spec: {
+    compressionMode: '',
+    replicated: {
+      size: 2,
+    },
+  },
+};
+
+export const inputProvisioner = $('button[id=storage-class-provisioner]');
+export const provisionerDropdown = $('#storage-class-provisioner');
+export const selectInput = $('[data-test-id=dropdown-text-filter]');
+export const createPoolDropdown = $('button[class=pf-c-dropdown__menu-item]');
+export const poolModal = $('modal-content');
+export const allowExpand = element(
+  by.cssContainingText('.checkbox', 'Allow persistent volume claims to be expanded'),
+);
+export const poolDropdown = 'ul.pf-c-dropdown__menu li';
```

Comment:
```suggestion
export const poolDropdown = $('.ul.pf-c-dropdown__menu li');
```
Doing this way, you don't need to use `protractor.By.css`